golang/cosmos/x/swingset/client/cli/tx.go

```diff
@@ -42,10 +42,34 @@ func GetTxCmd(storeKey string) *cobra.Command {
 	return swingsetTxCmd
 }
 
+// parseInput handles an input argument that may refer to a file.
+// If the input does not start with a @, it is returned as-is.
+// The slice after the @ represents the filename. If the filename is -
+// a single line from stdin is returned, otherwise the full content
```

Comment:
A single line? That seems sort of obscure. But I see code that did that before. Odd.

Reply:
Right. I did not change behavior in this PR, only extended behavior to an existing command